### PR TITLE
Clean up JSON Golden Tests in wire-api, Part #2

### PR DESCRIPTION
**Pull request by mdimjasevic**
This is the second PR in a series of PRs that clean up automatically generated JSON golden tests in `wire-api`.

## Checklist

 - [x] The **PR Title** explains the impact of the change.
 - [x] The **PR description** provides context as to why the change should occur and what the code contributes to that effect. This could also be a link to a JIRA ticket or a Github issue, if there is one.
 - [x] If end-points have been added or changed: the **endpoint / config-flag checklist** (see Wire-employee only backend [wiki page](https://github.com/zinfra/backend-wiki/wiki/Checklists)) has been followed.
 - [x] If a schema migration has been added, I ran **`make git-add-cassandra-schema`** to update the cassandra schema documentation.
 - [x] Section *Unreleased* of **CHANGELOG-draft.md** contains the following bits of information:
   - [x] A line with the title and number of the PR in one or more suitable sub-sections.
   - [x] If /a: measures to be taken by instance operators.
   - [x] If /a: list of cassandra migrations.
   - [x] If public end-points have been changed or added: does nginz need upgrade?
   - [x] If internal end-points have been added or changed: which services have to be deployed in a specific order?

### CODE REVIEW

File: CHANGELOG-draft.md [hunk 1]
```diff
@@ -42,5 +42,17 @@ THIS FILE ACCUMULATES THE RELEASE NOTES FOR THE UPCOMING RELEASE.
...
+* Make mapping between (team) permissions and roles more lenient (#1711)
+* The `DELETE /conversations/:cnv/members/:usr` endpoint rewritten to Servant (#1697)
+* Remove leftover auto-connect internal endpoint and code (#1716)
+* Clean up JSON golden tests (#1729, #1732)
+
+## Federation changes
+
+* Added client certificate support for server to server authentication (#1682)
+* Implemented full server-to-server authentication (#1687)
+* Add an endpoint for removing a qualified user from a local conversation (#1697)
```
Comment (akshaymankar) [inline CHANGELOG-draft.md:56]: This looks wrong

**Review by akshaymankar — DISMISSED** [review]

File: CHANGELOG-draft.md (hunk 1 above)
Comment (mdimjasevic) [reply CHANGELOG-draft.md:56]: Can you be a bit more specific?

Comment (akshaymankar) [reply CHANGELOG-draft.md]: Those "added" lines didn't look like they belonged to this PR, apologies for being vague. But it looks like you solved it :+1:

Comment (mdimjasevic) [reply CHANGELOG-draft.md]: Do you want to update your review or do I still need to make some changes?

Comment (akshaymankar) [reply CHANGELOG-draft.md]: I haven't looked at all the things, this was the only thing I found until the point I looked. I have dismissed the review. I will take a look once the conflicts are resolved.

Comment (mdimjasevic) [reply CHANGELOG-draft.md]: @akshaymankar , I just resolved the conflicts!

**Review by jschaul — APPROVED** [review]

File: CHANGELOG-draft.md (hunk 1 above)
Comment (mdimjasevic) [reply CHANGELOG-draft.md]: Marking this as resolved.

**Pull request merged**